src/styles/layout/sidebar.less

```diff
@@ -1,712 +1,47 @@
-
-.canvas-sidebar-open {
-
-  .sidebar {
-
-    left: 0px;
-
-  }
-
-  .page {
-
-    left: 100%;
-    margin-left: @sidebar-margin-offset * -1;
-    position: fixed;
-
-  }
-
-}
-
-
-
-
-
-
-& when (@screen-mini-enabled) {
-
-  /* -----------------------------------------------------------------------------
-  --------------------------------------------------------------------------------
-
-  Screen Size: Mini (@screen-mini) and above
-
-  --------------------------------------------------------------------------------
-  ----------------------------------------------------------------------------- */
-
-  @media (min-width: @screen-mini) {
-
-    /* -----------------------------------------------------------------------------
-    --------------------------------------------------------------------------------
-
-    Sidebar
-
-    --------------------------------------------------------------------------------
-    ----------------------------------------------------------------------------- */
-
-    .sidebar {
-
-      margin-left: @sidebar-margin-offset-screen-mini * -1;
-      padding-left: @sidebar-margin-offset-screen-mini;
-
-
-
-      /* -----------------------------------------------------------------------------
-      Sidebar Header
-      ----------------------------------------------------------------------------- */
-
-      .sidebar-header {
-
-
-
-        /* -----------------------------------------------------------------------------
-        Sidebar Header Label
-        ----------------------------------------------------------------------------- */
-
-        .sidebar-header-label {
-
-          display: none;
-
-        }
-
-
-
-        /* -----------------------------------------------------------------------------
-        Sidebar Header Sublabel
-        ----------------------------------------------------------------------------- */
-
-        .sidebar-header-sublabel {
-
-          display: none;
-
-        }
-
-      }
-
-
-
-      /* -----------------------------------------------------------------------------
-      Sidebar Content
-      ----------------------------------------------------------------------------- */
-      .sidebar-content {
-
-        /* -----------------------------------------------------------------------------
-        Sidebar Menu
-        ----------------------------------------------------------------------------- */
-
-        .sidebar-menu {
-
-          .sidebar-menu-item {
-
-            .sidebar-menu-item-icon {
-
-              margin-right: (@base-spacing-unit-screen-mini * 0.5);
-
-            }
-
-            a {
-
-              padding: (@base-spacing-unit-screen-mini * 0.5) (@grid-container-fluid-narrow-gutter-width-screen-mini * 1.0);
-
-            }
-
-          }
-
-        }
-
-      }
 
-    }
 
+.sidebar {
 
+  background: color-lighten(@neutral, -40);
+  box-shadow: 1px 0px 0px 0px hsla(hue(@neutral), saturation(@neutral), lightness(@neutral), 0.1);
```

Comment:
Well, this is clever... but In LESS you can write: `fade(@neutral, 10%)` which compiles to `rgba(#, #, #, 0.1)`